Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -1236,10 +1241,10 @@ eval (GetQuotesList (SelectListRes resp)) state = do
                 continue newState{props{selectedQuote = if value then Nothing else newState.props.selectedQuote}}
 
 eval (ContinueWithoutOffers (SelectListRes resp)) state = do
-  let bookingId = (fromMaybe "" resp.bookingId)
+  let bookingId = STR.trim (fromMaybe "" resp.bookingId)
```

Comment:
@ashishsingh101  can we remove fromMaybe here and use case statement

Reply:
👍